api/v1alpha1/clusterextension_types.go

```diff
@@ -511,22 +512,17 @@ type ClusterExtensionInstallStatus struct {
 	//
 	// A "bundle" is a versioned set of content that represents the resources that
 	// need to be applied to a cluster to install a package.
-	//
-	// This field is only updated once a bundle has been successfully installed and
-	// once set will only be updated when a new version of the bundle has
-	// successfully replaced the currently installed version.
-	//
-	//+optional
-	Bundle *BundleMetadata `json:"bundle,omitempty"`
+	Bundle BundleMetadata `json:"bundle"`
```

Comment:
I removed the pointers here because the parent struct is a pointer. If the parent struct is non-nil, this field is _always_ expected to be set.